testsuite/MDAnalysisTests/test_auxiliary.py

```diff
@@ -35,7 +35,12 @@ def __init__(self):
 class BaseAuxReaderTest(object):
     def __init__(self, reference):
         self.ref = reference
-        self.reader = self.ref.reader('test', self.ref.testdata)
+
+    def setUp(self):
```

Comment:
So `__init__` is when the class is created, and setUp is per test? Didn't know that